posts/2016-09-20-IDA2016-notes.md

```diff
@@ -0,0 +1,129 @@
+# International Data Week 2016 - Meeting Notes and Links
+
+*[International Data Week](internationaldataweek.org) in September 2016 brought together three events, [SciDataCon](http://www.scidatacon.org/), the International Data Forum, and the [Research Data Alliance 8th plenary](https://rd-alliance.org/plenaries/rda-eighth-plenary-meeting-denver-co). Joe Hand from the Dat team attended the conference and collected his notes here.*
+
+## SciDataCon2016
+
+SciDataCon featured sessions with 5-10 quick lightning talk each on a common theme. Notes from some of the more interesting sessions are below.
+
+### [Session on Sustainable Business Models for Data Repositories](http://www.scidatacon.org/2016/sessions/45/)
+
+These two sessions covered the costs, budgets, and revenue streams for various data repositories. There are many different pricing structures. Details on these can be found in the witness statements in the session link above. 
+
+Repositories are using a mix of subscriptions (paid by libraries, insitutions, researchers), pay to download (to get specific datasets), or pay to deposit (pay to upload to the repository), and external funding (government, private). 
+
+While each pricing model has its own strengths and downsides, it was clear the loser is the academic institutions, libraries, and researchers. Several library managers and researchers expressed concern that soon they will be unable to keep up financially with the number of data repository subscriptions they need to get data, similar to the issues being faced with journal subscriptions. Additionally, as more data becomes replicated in several places, it becomes unclear which repositories should be subscribed to.
```

Comment:
Perhaps the second sentence could be split up a little; found it hard to read. E.g something like this:
> Several library managers and researchers expressed concern that soon they will be unable to keep up financially with the number of data repository subscriptions they need in order to get data. These issues are similar to the ones being faced with journal subscriptions.